packages/-ember-data/package.json

```diff
@@ -68,67 +66,61 @@
     "silent-error": "^1.1.1"
   },
   "devDependencies": {
-    "@babel/plugin-transform-typescript": "^7.2.0",
-    "@ember-decorators/babel-transforms": "^5.2.0",
-    "@ember-decorators/data": "^5.1.4",
-    "@types/ember": "~3.0.29",
-    "@types/ember-qunit": "~3.4.5",
-    "@types/ember-test-helpers": "~1.0.5",
-    "@types/ember-testing-helpers": "~0.0.3",
-    "@types/ember__debug": "^3.0.3",
-    "@types/ember__test-helpers": "~0.7.8",
-    "@types/qunit": "^2.5.3",
-    "@types/rsvp": "^4.0.2",
-    "babel-eslint": "^10.0.1",
-    "broccoli-babel-transpiler": "^7.2.0",
-    "broccoli-concat": "^3.7.3",
-    "broccoli-stew": "^2.0.1",
-    "broccoli-string-replace": "^0.1.2",
-    "broccoli-test-helper": "^2.0.0",
-    "broccoli-uglify-sourcemap": "^3.0.0",
-    "co": "^4.6.0",
-    "common-tags": "^1.8.0",
-    "ember-cli": "^3.8.1",
-    "ember-cli-app-version": "^3.2.0",
-    "ember-cli-blueprint-test-helpers": "^0.19.1",
-    "ember-cli-dependency-checker": "^3.1.0",
-    "ember-cli-htmlbars": "^3.0.1",
-    "ember-cli-htmlbars-inline-precompile": "^2.0.0",
-    "ember-cli-inject-live-reload": "^2.0.1",
-    "ember-cli-internal-test-helpers": "^0.9.1",
-    "ember-cli-pretender": "^3.1.1",
-    "ember-cli-shims": "^1.2.0",
-    "ember-cli-sri": "^2.1.1",
-    "ember-cli-test-loader": "^2.2.0",
-    "ember-cli-typescript-blueprints": "^2.0.0-beta.1",
-    "ember-cli-uglify": "2.1.0",
-    "ember-cli-yuidoc": "^0.8.8",
-    "ember-decorators": "^5.2.0",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^2.0.0",
-    "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^4.4.1",
-    "ember-qunit-assert-helpers": "^0.2.2",
-    "ember-resolver": "^5.0.1",
-    "ember-source": "~3.8.0",
-    "ember-source-channel-url": "^1.1.0",
-    "ember-try": "^1.1.0",
-    "eslint": "^5.15.1",
-    "eslint-config-prettier": "^4.1.0",
-    "eslint-plugin-node": "^8.0.0",
-    "eslint-plugin-prettier": "^3.0.1",
-    "execa": "^1.0.0",
-    "github": "^1.1.1",
-    "glob": "^7.1.3",
-    "json-typescript": "^1.1.0",
-    "loader.js": "^4.7.0",
-    "mocha": "^6.1.1",
-    "mocha-only-detector": "1.0.0",
-    "prettier": "^1.15.3",
-    "rimraf": "^2.6.2",
-    "rsvp": "^4.8.4",
-    "typescript": "~3.4.2"
+    "@babel/plugin-transform-typescript": "*",
```

Comment:
I’m not 100% sure what yarn will do here (with all of these *’s). This seems fine for now, but keep an eye out for issues that present like the wrong version of a devDep.